src/main/java/baritone/behavior/PathingBehavior.java

```diff
@@ -468,6 +483,8 @@ private void findPathInNewThread(final BlockPos start, final boolean talkAboutIt
                         if (executor.get().getPath().positions().contains(expectedSegmentStart)) {
                             queuePathEvent(PathEvent.CALC_FINISHED_NOW_EXECUTING);
                             current = executor.get();
+                            ticksElapsedSoFar = 0;
```

Comment:
Perhaps these should be moved to a function like "resetEstimate", and maybe it should also be called when the goal changes? I'm not sure though, that isn't really necessary / might be too much work.

Reply:
lmk if you think this is reasonable, or too much work for not much benefit

Reply:
It would be nice if it was called again if the goal changes

Reply:
Yes but only if we start a full new path for that goal (e.g. not when the `GoalBlock` to `GoalXZ` simplification ends or `MineProcess` removes an ore). Would be nice if you tell me where to put the reset(s) to achieve that behaviour.

Reply:
yea i have no idea lol

Reply:
Is it an issue that this happens in a separate thread?